### ISSUE TITLE
Пропозиція переробити коментарі

### ISSUE THREAD

**Issue by uaxpasha** [hurtom/toloka#79]
Забагато місця уходить на один рядок коментаря, було б добре відійти від стандартних коментарів торент сайтів, і зробити їх більш компактними, наприклад ближче до редіта. Прибрати аватари, коли користувач був зареєстрований, останній раз мережі та все інше. І щоб відповіді до коментаря були під цим же коментарем  (треди). 

Верхній банер теж сподіваюсь переробите. Дякую

**Comment by yukoff** [comment]
Так, коментарі, як і весь дизайн/компонування будуть ~~перероблені~~ нові. #2

**Comment by tarasyyyk** [comment]
а мені подобаються такі коментарі, як тут: в Git'і, а саме – поділ на 2 вкладки Write i Preview. Дуже зручно. Можна ще додати оцінювання коментарів як на YouTube. Ну і зовсім шалена ідея – ajax завантаження коментів за потреби: `scroll to buttom` або простий `клік на кнопку`

**Comment by yukoff** [comment]
@taras96, так, дві вкладки для write/preview - зручно, Quire, наприклад, теж схожий підхід використовують. Десь такі коментарі і плануються, можливо і "Заванатажити ще" буде - тільки треба буде подумати, як з пагінацією бути.

**Comment by chief-j** [comment]
Я проти подібної форми коментарів на форумі
Цей стиль абсолютно не призначений для пересічних користувачів інтернету
"Нащо винаходити велосипед"?? Є готові рішення, такі як wysibb, sceditor, та інші. Вони є максимально зручними і функціональними.
Щодо ajax то я підтримую думку @yukoff . Один користувач надіслав повідомлення в базу, звідти до інших користувачів сповіщення "Завантажити ще", чи "нові коментарі". Здається, є подібне на xenforo, або розгирення для нього. Я вважаю, що це буде зручніше тим, що не всі заходять в тему, щоб поспілкуватися, а почитати заголовок

Дубль #58 , чи для обговорення?
:)

**Comment by yukoff** [comment]
> Цей стиль абсолютно не призначений для пересічних користувачів інтернету

Яку ж цікаву маячню іноді можна прочитати ;) І чим він (принципово) відрізняється від того, що зараз є? Тільки по кнопелю "Перегляд" не перезавантажується сторінка цілком, а ajax-ом завантажується прев'ю. Елементи форматування - майже повністю відповідають тим, що зараз. І головне - чим принципово воно відрізняється від того ж wysibb?

А про який, власне, "стиль" взагалі мова?

**Comment by yukoff** [comment]
Щодо #58 - ні, не дубль. Там мова про редактор, тут - про стрічку коментарів як таку.

**Comment by chief-j** [comment]
> чим принципово воно відрізняється від того ж wysibb?

А мова йде про "принципово відрізняється"? Мова йде про практичність, друже

**Comment by yukoff** [comment]
@chief-j Ну так практичність у MD (markdown) значно вище, ніж знімати руку з клавіатури, брати мишу, вести до іконки, тицяти, знову до клавіатири - поправити, що треба... Якщо про це була мова, бо що таке "практичність" у вашому розумінні - не дуже зрозуміло з попередніх коментарів

**Comment by chief-j** [comment]
@yukoff Є комбінації клавіш

**Comment by yukoff** [comment]
@chief-j Звісно, є - я ж до того, що мова йшла про _"пересічних користувачів інтернету"_, які часто-густо їх не знають.

**Comment by chief-j** [comment]
@yukoff не неси маячню. Ті самі "перпсічні" не захочуть вчити вчити команди і комбінації. Переважна більшість моїх друзів та знайомих (школа, універ, робота), друкували б мишею на екранній клавіатурі, якби знати, що така існує

**Comment by yukoff** [comment]
@chief-j Перечитай, будь ласка, що саме я написав, і чим воно відрізняється від твого
> Ті самі "перпсічні" не захочуть вчити вчити команди і комбінації

Це взагалі не смішно. Саме **ви** згадуєте про комбінації клавіш, на що я резонно відповідаю - "пересічні користувачі" їх здебільшого не знають, но що отримую випад, типу не неси маячні, вони не захочуть їх вчити... Логіка геть загубилася - сподіваюсь далі пояснювати не треба, який вийшов сюр з вашого "повчання"? ;)

**Comment by chief-j** [comment]
@yukoff пробач. То не до попереднього коментаря

**Comment by yukoff** [comment]
@chief-j Та я не ображаюся, ще чого ;) А от щодо
> Переважна більшість моїх друзів та знайомих (школа, універ, робота), друкували б мишею на екранній клавіатурі, якби знати, що така існує

то так, тенденція не надто радісна... Ще, здається, багато хто (мова не про твоїх знайомих, а взагалі) мізки б десь поклали, щоб думки всілякі не лізли (сумний смайлик)... І така тенденція у всьому світі спостерігається - така собі дебілізація :(

**Comment by chief-j** [comment]
Якими мої друзі були без розумі в школі (в той час без смартфонів та пк), такими й лишилися (вже із .. ). Це абсолютно нічого не змінило.
В дечому я кращий за них, в дечому вони кращі за мене. В результаті, загальної різниці між нами немає


А тепер по темі

> Забагато місця уходить на один рядок коментаря, було б добре відійти від стандартних коментарів торент сайтів, і зробити їх більш компактними, 

З цим я погоджуюся. Треба відійти від табличного стандарту коментарів трекера

> наприклад ближче до реддіта. прибрати аватари,, 

Аватари приберати не можна, категорично. Сторінка стане одноманітною

> відповіді до коментаря будуть під цим же коментарем,

Можливо було б зручно але і ні. Доведеться перескакувати із сторінки на сторінку, щоб прочитати останній коментар

>  від коли користувач на сайті

Зробити, як запропонував @yukoff .При наведенні з'являється вся необхідна інформація (або при кліку на аватар користувача)

> , щоб не писалось 2 рази "підтримка і поширення"(там пишеться звідки він + картинка з тим же написом)

Це фіча

> ,в мережі чи ні, і т.д.

Це не несе великої користі але не заважає. Можна зробити індикатор

**Comment by yukoff** [comment]
На ескізі у верхньому правому, так би мовити, куті аватарки показується червоний кружечок, якщо не в мережі, зелений - якщо в мережі.

**Comment by HereHaveSomeHelp** [comment]
@uaxpasha підтримую. 
Дуже бісить зовнішній вигляд комментарів на толоці. 
... давно тут нікого не було...
